ai2thor_wrapper/config_example.ini

```
@@ -0,0 +1,5 @@
+[ENV_SPECIFIC]
+;PICKUPABLE_OBJECTS=Mug
```

Modify the commented line as well because it is confusing otherwise
```suggestion
;PICKUP_OBJECTS=Mug
```